package.json

```diff
@@ -4,11 +4,11 @@
   "description": "",
```

Comment:
Shouldn't we consider adding description here?

Reply:
I'll do that, 👍🏽

Reply:
Done